tests/constants.py

```diff
@@ -141,6 +141,7 @@
 MULTICLASS_LABEL_SPACES = "iris_with_spaces_full"
 TEXT_GENERATION = "textgeneration"
 GEO_POINT = "geopoint"
+VECTOR_DATABASE = "vector_database"
```

Comment:
nit to be consistent with enum:
```suggestion
VECTOR_DATABASE = "vectordatabase"
```